packages/translate/test/index.js

```diff
@@ -462,44 +497,83 @@ describe('Translate', function() {
   });
 
   describe('request', function() {
-    it('should make the correct request', function(done) {
-      var userAgent = 'user-agent/0.0.0';
-
-      var getUserAgentFn = fakeUtil.getUserAgentFromPackageJson;
-      fakeUtil.getUserAgentFromPackageJson = function(packageJson) {
-        fakeUtil.getUserAgentFromPackageJson = getUserAgentFn;
-        assert.deepEqual(packageJson, require('../package.json'));
-        return userAgent;
-      };
+    describe('OAuth mode', function() {
+      var request;
 
-      var reqOpts = {
-        uri: '/test',
-        a: 'b',
-        c: 'd',
-        qs: {
+      beforeEach(function() {
```

Comment:
Actually there's an issue with the order of execution of `before` hooks, so we start seeing errors when using it.